flatdata-rs/lib/src/generator.rs

```diff
@@ -127,10 +127,19 @@ pub fn generate(
             .spawn()
             .map_err(|e| GeneratorError::Failure {
                 schema: entry.path().into(),
-                destination: result,
+                destination: result.clone(),
                 error: e,
             })?
-            .wait()?;
+            .wait()?
+            .success();
```

Comment:
You could use `code` here and put it into the error description below. This would provide more information in the error case.

Reply:
Done